com.unity.render-pipelines.universal/Runtime/XR/XRPass.cs

```diff
@@ -38,6 +38,7 @@ internal struct XRView
         internal readonly Rect viewport;
         internal readonly Mesh occlusionMesh;
         internal readonly int textureArraySlice;
+        internal readonly Vector2 eyeCenterUV;
```

Comment:
Quick note : this will create a merge conflict with https://github.com/Unity-Technologies/Graphics/pull/3557
Not a huge deal but let's discuss the order of PRs.